frontend/templates/product/sections/ProductSection/index.tsx

```diff
@@ -73,6 +73,19 @@ export function ProductSection({
 
    const isForSale = useIsProductForSale(product);
 
+   const compatibilityDrawerModelsTruncate = 4;
+   const compatibilityDrawerDeviceTruncate = 3;
+   const compatibilityDrawerIncomplete =
+      product.compatibility &&
+      (product.compatibility.devices.length >
+         compatibilityDrawerDeviceTruncate ||
+         !product.compatibility.devices
+            .slice(0, compatibilityDrawerDeviceTruncate)
+            .every(
```

Comment:
Why `.every()` and the reverse boolean over `.some(... length > ModelsTruncate)` ? Seems harder to read.

Reply:
uh i missed that function when looking over the docs. Yeah thats much better